x-pack/plugins/index_lifecycle_management/public/application/sections/policy_table/policy_table.tsx

```diff
@@ -85,8 +85,10 @@ export const PolicyTable: React.FunctionComponent<Props> = ({
       );
     }
 
+    // Wrapping the actual contents inside a div, otherwise the table layout will get messed up
```

Comment:
Found this bug while manually testing the apps we migrated, so decided to also patch it up real quick in this PR since it was such a small change. 

https://user-images.githubusercontent.com/1191206/123261155-aded8580-d4f6-11eb-923c-4e85528c3db0.mov